File: SisifoNotes.Lib/Models/Note.cs
using SisifoNotes.Lib.Core;
using System;
using System.Collections.Generic;
using System.Text;

namespace SisifoNotes.Lib.Models
{
    public class Note : Entity
    {
        public string Title { get; set; }
        public Guid ClientId { get; set; }
    }
}

File: SisifoNotes.Web/wwwroot/Scripts/Models/Note.js
class Note extends Entity
{
    constructor(json)
    {
        super(json)
        if (json)
        {
            this.Title = json.title;
            this.ClientId = json.clientId;
        }
        else
        {
            this.Title = "";
            this.ClientId = "00000000-0000-0000-0000-000000000000"
        }
    }
}